sunpy/net/tests/test_fido.py

```diff
@@ -68,7 +68,7 @@ def test_offline_fido(query):
 # Until we get more mocked, we can't really do this to online clients.
 # TODO: Hypothesis this again
 @pytest.mark.parametrize("query", [
-    (a.Instrument.eve & a.Time('2014/7/7', '2014/7/14') & a.Level.zero),
+    (a.Instrument.eve & a.Time('2014/7/7', '2014/7/14') & a.Level('0CS')),
```

Comment:
```suggestion
    (a.Instrument.eve & a.Time('2014/7/7', '2014/7/14') & a.Level('0CS')),
```
How has `0CS` been processed by the attrs registration code.